extensions/amp-ad/validator-amp-ad.protoascii

```diff
@@ -77,6 +83,40 @@ tags: {  # <amp-ad>
     supported_layouts: RESPONSIVE
   }
 }
+tags: {  # <amp-ad type="custom">
+  html_format: AMP  # Ads are not allowed inside ads
+  tag_name: "AMP-AD"
+  spec_name: "amp-ad with type=custom"
+  requires_extension: "amp-ad"  # See note at top of file
+  also_requires_tag_warning: "amp-ad extension .js script"
+  # If modifying disallowed_ancestors, then please also edit
+  # extensions/amp-auto-ads/*/placement.js
+  disallowed_ancestor: "AMP-APP-BANNER"
+  attrs: {
+    name: "data-uri"
```

Comment:
nit: `data-url`

Reply:
cc @lannka Do you expect this to break any current usage of custom ad? I think it's fine since we document that `data-url` is a mandatory attribute.

Reply:
There are two new requirements if `type=custom`

1) `data-url` is present
2) `data-url` value is `https`

Are you confident that both of those are sufficiently followed in the wild.

Reply:
No, but the docs always said it had to be an HTTPS url.